test/SlowTests/Tools/SetupSecuredClusterUsingRvn.cs

```diff
@@ -62,9 +62,6 @@ public async Task Should_Create_Secured_Cluster_Generating_Self_Singed_Cert_And_
             }
         };
         
-        Debug.Assert(setupInfo.ModifyLocalServer, nameof(setupInfo.ModifyLocalServer) + " == true");
```

Comment:
Does it mean it should be false after all? Do we check that later on in the tests if it modified the local server or not?

Reply:
It will be true if we are in the flow of the `SetupWizard`
Other than that we don't need to modify the server settings (`RVN`).
IIRC. That's what we have been talking about. So It won't be mandatory, no?

Reply:
Yeah, so we can assert that's `false` in the test probably?

Reply:
I see, np. Will push it in a few..

Reply:
I thought that you'd use regular XUnit assertion method `Assert.True()` to do those? If it's `Debug.Assert()` it's not gonna show up in Release if there's an issue and you can pass a message there too.